crates/bevy_diagnostic/src/diagnostic.rs

```diff
@@ -28,6 +28,30 @@ pub struct DiagnosticMeasurement {
     pub value: f64,
 }
 
+/// Enum for display percision. Choose `IntegerValue` for discrete values and `DecimalValue(num_of_decimals)` to specify the number of decimals
+#[derive(Debug)]
+pub enum DisplayPercision {
+    IntegerValue,
+    DecimalValue(usize),
+}
+
+impl DisplayPercision {
+    pub fn get_num_of_decimals(&self) -> usize {
+        match self {
+            DisplayPercision::IntegerValue => 0,
+            DisplayPercision::DecimalValue(x) => *x,
+        }
```

Comment:
This indicates that making an enum is redundant, as you can just use `usize` directly here.